focus-group.js

```diff
@@ -1,6 +1,6 @@
-export function menuKeyUX(options) {
```

Comment:
Can you also update `size-limit` config in `package.json` to see the cost of the changes?

Just update import, call `pnpm size-limit` and update the limit.